common/test/acceptance/pages/studio/settings_advanced.py

```diff
@@ -34,6 +37,40 @@ def _is_finished_loading():
         EmptyPromise(_is_finished_loading, 'Finished rendering the advanced policy items.').fulfill()
         return self.q(css='body.advanced').present
 
+    def get_key_names(self):
+        """
+            Returns a list of key names of all settings.
+        """
+        return self.q(css=KEY_CSS).text
+
+    def get_deprecated_settings_button_text(self):
+        """
+            Returns text for deprecated settings button
+        """
+        return self.q(css=DEPRECATED_SETTINGS_BUTTON_SELECTOR).text[0]
+
+    def click_deprecated_settings_button(self):
+        """
+            Show deprecated Settings
+        """
+        self.q(css=DEPRECATED_SETTINGS_BUTTON_SELECTOR).click()
+        if self.get_deprecated_settings_button_text() == 'Show Deprecated Settings':
+            self.wait_for_element_presence(DEPRECATED_SETTINGS_SELECTOR, 'Deprecated Settings are not present')
+        else:
+            self.wait_for_element_absence(DEPRECATED_SETTINGS_SELECTOR, 'Deprecated Settings are present')
+
+    def check_deprecated_settings_presence(self):
+        """
+            Returns true if deprecated settings are visible
+        """
+        return self.q(css=DEPRECATED_SETTINGS_SELECTOR).visible
+
+    def get_confirmation_message(self):
```

Comment:
Nit: Would this be better as a `@property` named `confirmation_message`?

Reply:
Same goes with get_confirmation_message, I will update it in the next PR, when I will add more tests.